Model/Api/CreateOrder.php

```diff
@@ -192,21 +197,24 @@ public function execute(
 
             $this->preProcessWebhook($immutableQuote->getStoreId());
 
-            $payload = $this->request->getContent();
             $transaction = json_decode($payload);
 
             /** @var Quote $quote */
             $quote = $this->orderHelper->prepareQuote($immutableQuote, $transaction);
-            $this->validateQuoteData($quote, $transaction);
 
-            /** @var \Magento\Sales\Model\Order $createOrderData */
-            $createOrderData = $this->orderHelper->preAuthCreateOrder($quote, $transaction);
```

Comment:
preAuthCreateOrder was split into 2 methods, processExistingOrder and processNewOrder, with validateQuoteData put in between them